kolibri/core/assets/src/utils/browserInfo.js

```diff
@@ -70,6 +70,16 @@ export const isTouchDevice =
   window.navigator?.maxTouchPoints > 0 ||
   window.navigator?.msMaxTouchPoints > 0;
 
+const device = info.device;
+
+// this is better than undefined
+device.type = device.type || 'unknown';
+device.model = device.model || 'unknown';
+device.vendor = device.vendor || 'unknown';
+export const deviceWithTouch = {
```

Comment:
I this would be superfluous! 
I think it would be more straight forward to just export `device` info only. I dont see the need to bundle it up with the `device` info.

I think the below would suffice
```js
export const device = info.device;
```

Reply:
Also `deviceWithTouch` could be misinterpreted to mean that it represents an object with touch capabilities which is not really the case -- `isTouchDevice` is a runtime value.